### ISSUE TITLE
The training hyper-parameters

### ISSUE THREAD

**Issue by Xinxiang7** [SystemErrorWang/White-box-Cartoonization#14]
Hello, Wang. I'm very interested in your job. I have two questions. Could you help me?
- Q1: 
I tried the training with λ1=1, λ2=10, λ3=λ4=2000, λ5=10000 as your paper suggests. But the result is not that good. I'm not sure it's the question of dataset or hyper-parameters?

- Q2: 
What's more, the hyper-parameters in the selective search is not sure. 
I tried seg_num=200, power=1.2, γ1=20, γ2=40 and the output of image was very black, as the following:
![200_1 2](https://user-images.githubusercontent.com/24989537/84728592-235d7680-afc4-11ea-8a21-c9f690383cd6.png)
 The result of simple search:
![ss_200](https://user-images.githubusercontent.com/24989537/84728893-f3fb3980-afc4-11ea-9c8b-b148597811ec.png)
After that, I tried seg_num=1500, power=0.35, γ1=20, γ2=40 and the output of image was not that bad:
![1500-0 35](https://user-images.githubusercontent.com/24989537/84729564-9bc53700-afc6-11ea-8fdd-b6fb92646770.png)

I know the value of pixel must be too large with power=1.2. So I just want to make sure that the parameters is suitable? 

Thanks very much.

**Comment by SystemErrorWang** [comment]
@Xinxiang7 maybe you can try to train the model with default parameters given in the code. for the selective search, we at first normalize the pixel values to (0, 1). I guess there may also be some problem in terminology (maybe 1.25 in my setting means 0.8 in your setting)

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks  a lot. I will try sooner.
I found the function of texture in the paper is different from the one in the code
In the paper:
![texture_1](https://user-images.githubusercontent.com/24989537/84730363-8ea94780-afc8-11ea-8646-ab33ec60b441.png)
the code:
`   output1 = (b_weight * b1 + g_weight * g1 + r_weight * r1) / (b_weight + g_weight + r_weight)
`
Does the standard grayscale image Y is not important. So It can be removed away or some other reason?

**Comment by SystemErrorWang** [comment]
@Xinxiang7 the code for random color shift is in utils.py and it strictly follows this formulation. please refer to it for details.

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks very much. 
I might be wrong. the code in utils.py as follows:
![test1](https://user-images.githubusercontent.com/24989537/84736351-35491480-afd8-11ea-875d-5c1c5809cd42.jpg)
I can't find the Y in the code. Does I miss something?

**Comment by SystemErrorWang** [comment]
@Xinxiang7 they are mathematically equivalent.

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks Very much for your reply!
